atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -92,10 +92,10 @@ private AtlasDbHttpClients() {
             AuxiliaryRemotingParameters clientParameters) {
         TargetFactory.InstanceAndVersion<T> fallbackProxy = fallbackProxySupplier.get();
         try {
-            return VersionSelectingClients.createVersionSelectingClientWithRefreshingNewClient(
+            return VersionSelectingClients.createVersionSelectingClient(
                     metricsManager,
-                    experimentalProxySupplier,
                     fallbackProxy,
+                    experimentalProxySupplier.get(),
```

Comment:
The ordering of the arguments has to be the other way (i.e. you still want `experimentalProxySupplier.get()` before `fallbackProxy`).